examples/online-shop/Dependencies.toml

```diff
@@ -0,0 +1,316 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
As discussed in the previous meeting, let's ignore Dependencies.toml file for examples

Reply:
removed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/fe7671fcf8c033cd93419840a549cb2dd8ad96d1